mne/io/tests/test_utils.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+"""Run tests for the utilities."""
+# Author: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#
+# License: BSD (3-clause)
+
+from ..utils import _check_orig_units
+
+
+def test_check_orig_units():
+    """Test the checking of original units."""
+    orig_units = dict(FC1='nV', Hfp3erz='n/a', Pz='uV', greekMu=u'μV',
```

Comment:
should we test for wrong capitalization? do we consider them valid or invalid?

Reply:
for now I suggest that we are lenient and accept any case. This can be changed once BIDS also employs stronger tests ... so far, the https://github.com/bids-standard/bids-validator/ does not check units for validity, so we are already a step ahead.